CMakeLists.txt

```diff
@@ -541,20 +324,29 @@ endif()
 
 if(ENABLE_LIBICONV)
   find_package(Iconv REQUIRED)
-  include_directories(SYSTEM ${Iconv_INCLUDE_DIRS})
 endif()
 
 if(ENABLE_LIBINTL)
   # LibIntl (not Intl) selects our FindLibIntl.cmake script. #8464
   find_package(LibIntl REQUIRED)
-  include_directories(SYSTEM ${LibIntl_INCLUDE_DIRS})
 endif()
 
 # Determine platform's threading library. Set CMAKE_THREAD_PREFER_PTHREAD
 # explicitly to indicate a strong preference for pthread.
 set(CMAKE_THREAD_PREFER_PTHREAD ON)
 find_package(Threads REQUIRED)
 
+set(NVIM_INCLUDE_LIBRARIES
+  ${Iconv_INCLUDE_DIRS}
+  ${LIBLUV_INCLUDE_DIRS}
+  ${LIBTERMKEY_INCLUDE_DIRS}
+  ${LIBUV_INCLUDE_DIRS}
+  ${LIBVTERM_INCLUDE_DIRS}
+  ${LibIntl_INCLUDE_DIRS}
```

Comment:
`${LibIntl_INCLUDE_DIRS}` will be defined only with `ENABLE_LIBINTL` (same with `${Iconv_INCLUDE_DIRS}`). Is including non-existent variables okay?

Reply:
Should be fine since non-existing variables evaluate to an empty string. However, maybe it's clearer to do

```cmake
if(ENABLE_LIBINTL)
  # LibIntl (not Intl) selects our FindLibIntl.cmake script. #8464
  find_package(LibIntl REQUIRED)
  target_include_directories(main_lib SYSTEM INTERFACE ${LibIntl_INCLUDE_DIRS})
  target_link_libraries(main_lib INTERFACE ${LibIntl_LIBRARY})
endif()
```

directly, rather than collect them in a variable?